CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Windows:
 Bug fixes:
 * Fix a bug with ``import_vars`` functionality which didn't work correctly when the same variable name prefix was used (e.g. ``SCALYR_FOO_TEST``, ``SCALYR_FOO``).
 * Fix a bug with handling the log file of the Kubernetes Event Monitor twice, which led to duplication in the agent's status. 
+* Fix a bug in scalyr-agent-2-config ``--export-config``, ``import-config`` options caused by Python 2 and 3 code incompatibility.
 
 Docker images:
 * Upgrade various dependencies: orjson, requests, zstandard, lz4, docker.
```

Comment:
Do we also need a changelog entry here for docker image? Aka something along the lines of "Make sure ``scalyr-agent-2-config`` binary is present in the Docker image``.

Reply:
Ah sure, ~was trying to hide my oversight~ forgot about it.